docs/cli-reference.rst

```diff
@@ -35,6 +35,18 @@ include the minor version (e.g. ``… python3.6``) – use what fits your circum
 On Windows, the Python launcher ``py`` knows how to handle shebangs using ``env``,
 so it's overall the best choice if you target multiple platforms with a pure Python zipapp.
 
+If you have ``coreutils`` with at least version 8.30 on your Linux system,
+then you can use env shebangs *and* pass arguments to Python, like in this example:
+
+.. code-block:: shell
+
+    shiv -p '/usr/bin/env -S python3 -I -S' …
+
+The magic ingredient is the new ``-S`` (split) option, but as of 2020
+you have to target very new stable releases like Debian Buster
+or rolling distributions for it to actually work.
+Check for the option by calling ``env --help``.
+
```

Comment:
Can we omit this entry for this change? I appreciate that we want to educate users about what is possible with shebangs, but given that the tool in question (shiv) has no opinion about the shebang, I'd rather not introduce this concept in our documentation.